### PR TITLE
add skiped urls

### PR DESCRIPTION
**Pull request by olegtrofimenko**

### CODE REVIEW

File: src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs [hunk 1]
```diff
@@ -31,6 +47,12 @@ public CheckBrowserMiddleware(RequestDelegate next, IEnumerable<Browser> browser
         /// <returns></returns>
         public Task Invoke(HttpContext context)
         {
+            if (context.Request.Path.HasValue && _skipUrls.Contains(context.Request.Path.Value.ToLower()))
```
Comment (MikhailGerasimov) [inline src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs]: Should we check all values on Null?

File: tests/Lykke.Backoffice.Common.Tests/CheckBrowserTest.cs [hunk 2]
```diff
@@ -49,7 +49,12 @@ public CheckBrowserTest()
                 MaxMajorVersion = "20"
             };
             list.Add(browser);
-            _middleware = new CheckBrowserMiddleware(RequestDelegateMock.Object.RequestDelegate, list);
+
+            var skipUrls = new List<string>();
```
Comment (MikhailGerasimov) [inline tests/Lykke.Backoffice.Common.Tests/CheckBrowserTest.cs]: Use array initialization

File: src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs [hunk 3]
```diff
@@ -23,6 +24,21 @@ public CheckBrowserMiddleware(RequestDelegate next, IEnumerable<Browser> browser
         {
             _next = next;
             _browsers = browsers;
+            var urls = new List<string>();
+            urls.Add("/api/isalive");
+            _skipUrls = urls;
+        }
+        /// <summary>
+        ///
```
Comment (MikhailGerasimov) [inline src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs]: Add a meaningful  description

File: src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs [hunk 4]
```diff
@@ -23,6 +24,21 @@ public CheckBrowserMiddleware(RequestDelegate next, IEnumerable<Browser> browser
         {
             _next = next;
             _browsers = browsers;
+            var urls = new List<string>();
+            urls.Add("/api/isalive");
```
Comment (MikhailGerasimov) [inline src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs]: Use array initialization

**Review by mskobzev-old — DISMISSED** [review]

**Review by mskobzev-old — APPROVED** [review]

**Review by MikhailGerasimov — APPROVED** [review]

**Pull request merged**